cmd/vcluster/cmd/snapshot.go

```diff
@@ -292,7 +292,7 @@ func startEmbeddedBackingStore(ctx context.Context, vConfig *config.VirtualClust
 				return fmt.Errorf("failed to create directory %s: %w", filepath.Dir(constants.K3sSqliteDatabase), err)
 			}
 
-			k8s.StartKine(ctx, fmt.Sprintf("sqlite://%s?_journal=WAL&cache=shared&_busy_timeout=30000", constants.K3sSqliteDatabase), constants.K3sKineEndpoint, nil)
+			k8s.StartKine(ctx, fmt.Sprintf("sqlite://%s?_journal=WAL&cache=shared&_busy_timeout=30000", constants.K3sSqliteDatabase), constants.K3sKineEndpoint, nil, nil)
```

Comment:
Should you maybe pass here kine extraArgs from config?

Reply:
They are not needed here since this basically should mock k3s and k3s does not pass any extra kine args